docs/pages/setup/reference/cli.mdx

```diff
@@ -1310,6 +1310,346 @@ $ tctl auth rotate --type=user --grace-period=200h
+      --secret SECRET --google-id GOOGLE_ID --google-acc-uri /var/lib/teleport/gacc.json \
+      --google-admin admin@mydomain.example.com
+```
+
+4. Generate the configuration and immediately test it using `tctl sso test`` command.
+
+```code
+$ tctl sso configure oidc ... | tctl sso test
+```
+
+
+### tctl sso configure saml
+
+Configure SAML auth connector, optionally using a preset.
+
+The flag `--attributes-to-roles/-r` can be provided multiple times. 
+
+To avoid errors when pasting XML to YAML file, we encourage the usage of flag `--entity-descriptor`/`-e`.
+
+```code
+$ tctl sso configure saml -e entity_desc.xml -r attr,value,role1 [<other-flags>]
+```
+
+#### Arguments
+
+This command accepts no arguments.
+
+#### Flags
+
+Mandatory flags: 
+- `--attributes-to-roles`; may be given multiple times.
+- `--entity-descriptor`, unless `--sso`, `--acs`, `--cert` and `--issuer` are also given.
+- `-n/--name`, unless `-p/--preset` is present.
+
+| Name                        | Description                                                                                                                                      |
+|-----------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------|
+| `-p/--preset`               | Preset. One of: `okta`, `onelogin`, `ad`, `adfs`.                                                                                                |
+| `-n/--name`                 | Connector name. Required, unless implied from preset.                                                                                            |
+| `-e/--entity-descriptor`    | Set the Entity Descriptor. Valid values: file, URL, XML content. Supplies configuration parameters as single XML instead of individual elements. |
+| `-r/--attributes-to-roles`  | Sets attribute-to-role mapping using format `attr_name,attr_value,role1,role2,...`. Repeatable.                                                  |
+| `--display`                 | Sets the connector display name.                                                                                                                 |
+| `--issuer`                  | Sets identity provider issuer.                                                                                                                   |
+| `--sso`                     | SSO is the URL of the identity provider's SSO service.                                                                                           |
+| `--cert`                    | Cert file with with the IdP certificate PEM. IdP signs responses using this certificate.                                                         |
+| `--acs`                     | AssertionConsumerService is a URL for assertion consumer service on the service provider (Teleport's side).                                      |
```

Comment:
```suggestion
| `--acs`                     | AssertionConsumerService is a URL for the assertion consumer service on the service provider (Teleport's side).                                      |
```
Should we also add a quick note about how a user would determine this from their Teleport cluster?